modules/storage/cosmos-db/modules/sql_containers.bicep

```diff
@@ -0,0 +1,74 @@
+param cosmosDBAccountName string
+param enableServerless bool = false
+param sqlDatabaseName string
+param sqlDatabaseContainer object
+
+resource cosmosDBAccount 'Microsoft.DocumentDB/databaseAccounts@2022-11-15' existing = {
```

Comment:
use nested resource syntax per other comment to reduce this template a little bit.

Reply:
Removed from all modules.